### DISCUSSION TITLE
Set interval of backup

### DISCUSSION THREAD

**Discussion by akshayparseja** [Altinity/clickhouse-backup#1021] [General]
How do i set interval of backup rather than doing every 24h i want once in a week backup.
using the operator and a container running how do we set it up

**Comment by Slach** [comment]
look to
https://github.com/Altinity/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes

modify `kind: CronJob` manifest
with different scehdule
which will do only full backup weekly

**Reply by akshayparseja** [reply]
I am running a container clickhouse-backup , the cron job itself is running clickhouse server for backup.
Can i not use the clickhouse-backup image and schedule it?

**Reply by Slach** [reply]
cronjob doesn't run clickhouse-server 
cronjob executes SQL commands via clickhouse-client and connect to exists clickhouse-server which deployed with clickhouse-operator via `kind: ClickHouseInstallation`

try to figure out with manifests from link which i shared above
